File: cs/brainz/Responses/ReleaseGroupResponse.cs
namespace brainz.Responses
{
    using brainz.Model;
    using Newtonsoft.Json;

    public partial class ReleaseGroupResponse
    {
        [JsonProperty("release-groups")]
        public ReleaseGroup[] ReleaseGroups { get; set; }

        [JsonProperty("release-group-count")]
        public long ReleaseGroupCount { get; set; }

        [JsonProperty("release-group-offset")]
        public long ReleaseGroupOffset { get; set; }
    }
}

File: go/src/responses/releasegroupresponse.go
package responses

import model "../model"

type ReleaseGroupResponse struct {
	ReleaseGroups      []model.ReleaseGroup `json:"release-groups"`
	ReleaseGroupCount  int                  `json:"release-group-count"`
	ReleaseGroupOffset int                  `json:"release-group-offset"`
}
